src/app/cdk/side-bar/modals/modal-email/modal-email.component.html

```diff
@@ -10,10 +10,91 @@
     </button>
   </app-modal-footer>
 
-  <p>ORCID works best when you have multiple email addresses.</p>
-  <p>
-    Per the ORCID terms of use you may only add email addresses that you have
-    control over. Each time you add an email address we will send you an email
-    asking you to verify it.
-  </p>
+  <div class="container" [formGroup]="emailsForm">
+    <div class="row">
+      <div class="col">
+        <p>ORCID works best when you have multiple email addresses.</p>
```

Comment:
@leomendoza123 All attributes i18n are missing in this file

Reply:
Thanks for noticing Dani. I will solve this as part of https://trello.com/c/4EOdjRy6/7214-speedup-bio-emails-5